backend/plugin/db/driver.go

```diff
@@ -78,6 +78,7 @@ type InstanceMetadata struct {
 	InstanceRoles []*storepb.InstanceRoleMetadata
 	// Simplified database metadata.
 	Databases []*storepb.DatabaseMetadata
+	Metadata  *storepb.InstanceMetadata
```

Comment:
Maybe `Version` can be in `MetaData` too?

Reply:
Yup, maybe later as not the highest priority.